SPEC.md

```diff
@@ -201,25 +201,170 @@ Note that logging mechanisms other than stdout and stderr are not required by th
 ### Isolators
 
 Isolators enforce resource constraints rather than namespacing.
-Isolators may be applied to individual applications, to whole containers, or to both.
+Isolators may be scoped to individual applications, to whole containers, or to both.
```

Comment:
I suppose it is OK, but "notifying the user" seems a bit vague. @cdaylward any better ideas?